lemminx-liberty/src/main/java/io/openliberty/tools/langserver/lemminx/LibertyHoverParticipant.java

```diff
@@ -47,6 +46,15 @@ public Hover onAttributeName(IHoverRequest request, CancelChecker cancelChecker)
 
     @Override
     public Hover onAttributeValue(IHoverRequest request, CancelChecker cancelChecker) {
+        List<String> variables = LibertyUtils.getVariablesFromTextContent(request.getNode().getTextContent());
+        if (variables.size() == 1) {
```

Comment:
Done. since hover can be only shown once for an attribute value, multiple variables will be appended to same hover as below. is this okay?

![image](https://github.com/user-attachments/assets/2010295c-ddbb-4831-93e8-d68ede9edcc6)

Reply:
I think that is fine. I would remove the "for variable" part though.